### PR TITLE
Update business-continuity-overview.md

### PR DESCRIPTION
**Pull request by cosh**

### CODE REVIEW

**Comment by PRMerger8** [comment]
@cosh : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@shsagir 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by YaelSchuster** [comment]
Approved by ADX content team
#sign-off

**Comment by PRMerger4** [comment]
@YaelSchuster: I'm sorry - only the author of this article, @shsagir, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: data-explorer/business-continuity-overview.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ High availability refers to the fault-tolerance of Azure Data Explorer, its comp
...
+Azure Data Explorer leverages Azure Storage as its durable persistence layer. Azure Storage automatically provides fault tolerance, with the default setting offering Locally Redundant Storage (LRS) within a data center. Three replicas are persisted. If a replica is lost while in use, another is deployed without disruption. Further resiliency is possible with Zone Redundant Storage (ZRS) that places replicas intelligently across Azure regional availability zones for maximum fault tolerance at an additional cost. ZRS enabled storage is automatically configured when the Azure Data Explorer cluster gets deployed into [Availability Zones](create-cluster-database-portal.md#create-a-cluster).
```
Comment (shsagir) [inline data-explorer/business-continuity-overview.md:44]:
```suggestion
Azure Data Explorer leverages Azure Storage as its durable persistence layer. Azure Storage automatically provides fault tolerance, with the default setting offering Locally Redundant Storage (LRS) within a data center. Three replicas are persisted. If a replica is lost while in use, another is deployed without disruption. Further resiliency is possible with Zone Redundant Storage (ZRS) that places replicas intelligently across Azure regional availability zones for maximum fault tolerance at an additional cost. ZRS enabled storage is automatically configured when the Azure Data Explorer cluster is deployed into [Availability Zones](create-cluster-database-portal.md#create-a-cluster).
```

**Comment by PRMerger5** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by shsagir** [comment]
#sign-off

**Comment by shsagir** [comment]
Fixes #https://github.com/MicrosoftDocs/dataexplorer-docs/issues/1215

**Pull request merged**